src/opm/output/eclipse/AggregateNetworkData.cpp

```diff
@@ -447,6 +448,9 @@ void staticContrib(const Opm::Schedule&     sched,
     if (sched.hasGroup(nodeName, lookup_step)) {
         iNode[Ix::Group] = sched.getGroup(nodeName, lookup_step).insert_index();
     }
+    if (nodeName == "FIELD") {
```

Comment:
@bska, do you have understanding about this line? Can you help to have a look? Thanks.